pandas/core/frame.py

```diff
@@ -115,6 +115,13 @@
     side, respectively
 copy : boolean, default True
     If False, do not copy data unnecessarily
+indicator : boolean, default False
```

Comment:
I suppose you could allow this to be a string as well, which would be the *name* of the resultant indicator column (instead of ``_merge``). thoughts?